### ISSUE TITLE
Experimental setup for computing precision/recall

### ISSUE THREAD

**Issue by Badr-MOUFAD** [kynkaat/guidance-interval#4]
Many thank you for this work

About the computation of precision/recall (figure 4),

I would be grateful if you could confirm that the experimental setups follows [1] and that
- **reference dataset**: ImagenetNet validation set
- **image representation**: computed with Dinov2
- **number of samples**: 10k
- **number of neighbors**: 5

I appreciate your time 🙏 

also pinging @YazidJanati

---
.. [1] [Exposing flaws of generative model evaluation metrics and their unfair treatment of diffusion models](https://arxiv.org/abs/2306.04675)

**Comment by kynkaat** [comment]
Hi,

Thanks for your clarifying question. The experimental setup follows [1] only in the sense that we use the feature space of the DINOv2 model. For other parameters, we used the ImageNet training set, 50k real and generated samples and number of neighbors `k = 3`.

**Comment by Badr-MOUFAD** [comment]
Thanks for the clarification 🙏 
I appreciate your time and prompt response

**Comment by kynkaat** [comment]
No worries!
